### ISSUE TITLE
Response time by http method prometheus metric

### ISSUE THREAD

**Issue by clwluvw** [haproxy/haproxy#580]
## What should haproxy do differently? Which functionality do you think we should add?

Add http response time with http method in prometheus exporter.

## What are you trying to do?

Monitor the response time of my http requests by http methods.

**Comment by roidelapluie** [comment]
See https://github.com/haproxy/haproxy/issues/398, I guess response would be a label of those histograms?

**Comment by clwluvw** [comment]
> See #398, I guess response would be a label of those histograms?

Yes that is what I need, too. HTTP methods should be a label of those histograms. Would you please backport for 2.0.x when it gets done?

**Comment by capflam** [comment]
@clwluvw, feature requests must be discussed before thinking to backport anything anywhere. About histograms, I guess Willy's comment to #398 is quite clear :)
